tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -15,39 +17,67 @@
-            <converters:EmptyStringToObjectConverter x:Key="EmptyStringToObjectConverter" NotEmptyValue="Visible" EmptyValue="Collapsed"/>
+            <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" TrueValue="Visible" FalseValue="Collapsed"/>
+            <converters:BoolToVisibilityConverter x:Key="NegatedBoolToVisibilityConverter" TrueValue="Collapsed" FalseValue="Visible"/>
 
             <!--- Template for dev drive horizontal cards on the dev insights page. -->
             <DataTemplate x:Key="HorizontalCardForDevDriveInsightsPage" x:DataType="devViewModels:DevDriveCardViewModel">
-                <ctControls:SettingsCard Header="{Binding DevDriveLabel, Mode=OneWay}">
-                    <ctControls:SettingsCard.HeaderIcon>
-                        <FontIcon Glyph="&#xEDA2;" FontSize="25" FontFamily="{ThemeResource AmcFluentIcons}"/>
-                    </ctControls:SettingsCard.HeaderIcon>
-                    <Grid Grid.Column="2">
-                        <Grid.RowDefinitions>
-                            <RowDefinition Height="Auto"  />
-                            <RowDefinition Height="Auto" />
-                            <RowDefinition Height="Auto" />
-                        </Grid.RowDefinitions>
-                        <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 115 5">
-                                <Run Text="{Binding Path=DevDriveSizeText, Mode=OneWay}" />
-                        </TextBlock>
-                        <ProgressBar Grid.Row="1" Width="200" Value="{x:Bind DevDriveFillPercentage, Mode=OneWay}" HorizontalAlignment="Right" />
-                        <Grid Grid.Row="2">
-                            <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 120 0">
-                                    <Run Text="{Binding Path=DevDriveUsedSizeText, Mode=OneWay}" />
-                            </TextBlock>
-                            <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right">
-                                    <Run Text="{Binding Path=DevDriveFreeSizeText, Mode=OneWay}" />
-                            </TextBlock>
-                        </Grid>
-                    </Grid>
-                </ctControls:SettingsCard>
+                <StackPanel Orientation="Vertical">
+                    <ctControls:SettingsCard Header="{Binding DevDriveLabel, Mode=OneWay}" Visibility="{Binding Path=IsDevDriveTrusted, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}">
+                        <ctControls:SettingsCard.Description>
+                            <TextBlock Text="{Binding Path=DevDriveTrustText, Mode=OneWay}" Foreground="{ThemeResource SystemFillColorSuccessBrush}"/>
```

Comment:
DevDriveTrustText no longer needs to change depending on IsDevDriveTrusted. Reference Uid and remove the Property.